modules/redis/redis.c

```diff
@@ -45,14 +47,13 @@ typedef struct
   LogTemplateOptions template_options;
 
   GString *command;
-  LogTemplate *key;
-  GString *key_str;
-  LogTemplate *param1;
-  GString *param1_str;
-  LogTemplate *param2;
-  GString *param2_str;
+  GList *fields;
```

Comment:
[optional] instead of `fields` I would prefer `arguments`.

Reply:
Thanks, good idea, I changed it.